document the need for different instance names if CWA=true (#835)
kiali/kiali.io: content/en/docs/Configuration/namespace-management.md

By default, the Kiali Server is given cluster-wide access to all namespaces on the local cluster. This is controlled by the Kiali CR setting `deployment.cluster_wide_access`, which has a default value of `true` when not specified.

{{% alert color="info" %}}

> You cannot have multiple Kiali Servers with cluster wide access that have identical instance names. If you wish to install multiple Kiali Servers with cluster wide access enabled, they each must have unique `deployment.instance_name` values.

Reviewer: ```suggestion
You cannot have multiple Kiali Servers with both cluster-wide access and identical instance names. If you wish to install multiple Kiali Servers with cluster-wide access enabled, each must have a unique `deployment.instance_name` value.
```
